conda-forge.yml

```diff
@@ -13,4 +13,3 @@ provider:
   linux: azure
   osx: azure
   win: azure
-upload_on_branch: v1.5.x
```

Comment:
I don't think we should remove it. Could you please revert it?

Reply:
reverted

Reply:
Thanks!